src/pages/myNotes/note/note.jsx

```diff
@@ -0,0 +1,40 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import { Box } from '@material-ui/core';
+import useStyles from '../styled';
+
+const Note = ({ title, description, date, changeState, id, notActive, changeText }) => {
+  const classes = useStyles();
+  const cutSymbols = () => {
+    let str = '';
+    if (description.length > 20) {
+      str = description.substring(0, 20);
+      return (`${str}...`);
+    }
+    return description;
+  };
+  return (
+    <Box
+      className={notActive === id ? classes.activeNote : classes.note}
+      onClick={() => {
+        changeState(id);
+        changeText(title, description, date);
```

Comment:
We can call here just `setActiveNote(id)`, and inside the `MyNotes` component find active note from collection and send it inside `Note`.